modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -111,7 +111,7 @@ class Instrument_Manager extends \NDB_Menu_Filter
                     " -p" . escapeshellarg($db_config['quatPassword']).
                     " " . escapeshellarg($db_config['database']).
                     " < " . $this->path . "project/tables_sql/".
-                    $instrument->table.".sql"
+                    escapeshellarg($instrument->table).".sql"
```

Comment:
the closing bracket is wrong, the argument being escaped includes the ".sql" part of the filename.